src/utils/getAllReportees.js

```diff
@@ -31,15 +31,17 @@ const buildOrgChartData = (people) => {
   return org;
 };
 
-const getOrgChart = async (secretsClient) => {
-  const { secret } = await await secretsClient.get(config.taskclusterSecrets.orgData);
-  return buildOrgChartData(secret.employees);
-};
-
 const findReportees = (completeOrg, email) => {
   let allReportees = {};
-  allReportees[email] = completeOrg[email];
-  const { reportees } = completeOrg[email];
+
+  // if non-LDAP user, replace user email by the last email in list
+  const allEmails = Object.keys(completeOrg);
+  const checkedEmail = (email in completeOrg) ? email : allEmails[allEmails.length - 1];
```

Comment:
I don't understand why you're getting the last email in the list.
Could you be more explicit in your comment? Explain what is the case you're handling here, what are the conditions to be here...